BenchBnb/BenchBnb/Models/Bench.cs

```diff
@@ -19,6 +19,12 @@ public class Bench
         [Required, Column("Longitude")]
         public float Longitude { get; set; } 
 
+
+
+        public Bench()
+        {
+
+        }
         public Bench (string description, int numSeats, float latitude, float longitude)
```

Comment:
You might consider associating a bench with a user so you know who created it.

Reply:
Fixed